Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM lsiobase/alpine.nginx:3.6
+MAINTAINER christronyxyocum
+
+# Install packages
+RUN \
+ apk add --no-cache \
+	curl \
+	php7-curl \
+	php7-ldap \
+	php7-pdo_sqlite \
+	php7-sqlite3 \
+	php7-session \
+	php7-zip
+
+# Add local files
+COPY root/ /
+
+# Port and volumes
+EXPOSE 80
+VOLUME /config
```

Comment:
Expose `/var/log` too?

Reply:
I don't think it has to be exposed like /config. With these base images, you need to map /var/logs or they won't start. I just need to update the README and Docker hub page to reflect that.